src/projects/automation/workflows/workflow.md

```diff
@@ -0,0 +1,81 @@
+# Workflow Overview
+
+A Workflow comprises an orchestrated and repeatable pattern of activities. In Toca, the Workflow is the highest level of automation. It is the final stage where you put all your activities together to represent your business process. The workflow may be considered a view or representation of the real work.
+
+**Nodes**
+Nodes can be dragged onto your workflow canvas.
+- Start - Denotes the start point of your workflow. Can only be used once.
+- Exclusive - Defines a decision point in the workflow
+- Trigger - Allows you to start another workflow
+- label - A simple text label with customisable font options.
+- Stop - Denotes the point at which the automation should stop. A workflow can have multiple stop nodes.
+
+**Error handling**
```

Comment:
This section seems more geared towards activities.

Reply:
I know, but think it is worth talking about here

Reply:
in general terms yes, but you're describing minutiae that cannot be configured via the workflow UI.

Reply:
Okay, I concede, have moved this to the "Activity" page